package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "product-specifications",
+  "version": "0.8.2",
+  "description": "Specifications table for WooCommerce",
+  "author": "Dornaweb",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "product-specifications",
+    "woocommerce",
+    "specifications table"
+  ],
+  "homepage": "https://github.com/dornaweb/product-specifications",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dornaweb/product-specifications"
+  },
+  "bugs": {
+    "url": "https://github.com/dornaweb/product-specifications/issues"
+  },
+  "engines": {
+    "node": ">=18.0"
+  },
+  "devDependencies": {
+    "@playwright/test": "^1.46.1",
+    "@wordpress/scripts": "^28.6.0",
+    "autoprefixer": "^10.4.20",
+    "eslint": "^9.9.1",
+    "postcss": "^8.4.42",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "sass": "^1.77.8",
+    "typescript": "^5.5.4"
+  },
+  "scripts": {
+    "start": "wp-scripts start",
```

Comment:
if is possible add dev script because developers usually use this script to go development mode
actually we can use this to know the project mode is production or staging or development mode

Reply:
@mehdiraized The [`wp-scripts start`](https://github.com/WordPress/gutenberg/tree/trunk/packages/scripts#start) seems to be in development mode. and `wp-scripts build` builds assets in production mode.